From: Dermot Bradley <dermot_bradley@yahoo.com>
Date: Tue, 15 Nov 2022 18:08 +0000
Subject: [PATCH] Add hyperv PTP rule

This change has already been merged upstream in eudev as #218.

---

diff -aur a/rules/50-udev-default.rules b/rules/50-udev-default.rules
--- a/rules/50-udev-default.rules
+++ b/rules/50-udev-default.rules
@@ -84,4 +84,6 @@
 
 SUBSYSTEM=="ptp", ATTR{clock_name}=="KVM virtual PTP", SYMLINK += "ptp_kvm"
 
+SUBSYSTEM=="ptp", ATTR{clock_name}=="hyperv", SYMLINK += "ptp_hyperv"
+
 LABEL="default_end"
